src/components/text/stories/text.stories.mdx

```diff
@@ -324,3 +324,12 @@ export const TextExamples = weight => {
 **Weight**: 400
 
 <Typeset>{TextExamples(TEXT_WEIGHT.NORMAL)}</Typeset>
+
+## Accessibility
+
+Each component has its own accessibility documentation containing a set of rules and usage examples to help implement accessible and readable text.
```

Comment:
Doesn't accessible mean also readable?

Reply:
I actually was thinking about removing "readable" because "accessible" is broader and covers "readability" rules